autosklearn/pipeline/components/data_preprocessing/balancing/balancing.py

```diff
@@ -14,10 +14,11 @@
 
 
 class Balancing(AutoSklearnPreprocessingAlgorithm):
-    def __init__(self, strategy: str = 'none',
-                 random_state: Optional[np.random.RandomState] = None,):
+    def __init__(
+        self,
+        strategy: str = 'none',
```

Comment:
Shouldn't the random_state stay for having a consistent API?

Reply:
I was unsure about this as no balancing actually uses `random_state` but it probably makes sense to keep it and pass it. I'll reinclude it.